doc: updade README and make linter happy (#421)
babylonchain/babylon: README.md

> # Babylon

Reviewer: Wrap to 80 char lines?

Unlocking 21 Million ₿ to Secure the Decentralized Economy

[![Website](https://badgen.net/badge/icon/Website?label=)](https://babylonchain.io)
[![Twitter](https://badgen.net/badge/icon/twitter?icon=twitter&label)](https://twitter.com/babylon_chain)
[![Discord](https://badgen.net/badge/icon/discord?icon=discord&label)](https://discord.com/invite/babylonglobal)
[![Medium](https://badgen.net/badge/icon/medium?icon=medium&label)](https://medium.com/babylonchain-io)

> [Babylon](https://babylonchain.io) provides a suite of Bitcoin security-sharing protocols. Currently, it includes two protocols:

Reviewer: ```suggestion
[Babylon](https://babylonchain.io) provides a suite of security-sharing protocols between Bitcoin and the PoS world. It supports two inter-connected protocols:
```

> - **Bitcoin timestamping:** This protocol sends succinct and verifiable timestamps of any data (such as PoS blockchains) to Bitcoin.
> - **Bitcoin staking:** This protocol allows bitcoin the asset to provide economic security to any decentralized systems through trustless (and self-custodian) staking.

Reviewer: ```suggestion
- **Bitcoin timestamping:** Submits succinct and verifiable timestamps of any data (such as PoS blockchains) to Bitcoin.
- **Bitcoin staking:** Enables Bitcoin holders to provide economic security to any decentralized system through trustless (and self-custodian) staking.
```
